OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.h

```diff
@@ -45,25 +46,25 @@ namespace OpenSim {
     specifying these alternative frames, Moco does not know which force to use
     (the force on the sphere or the force on the half-space) when summing the
     contact forces across contact force elements.
-    @see MocoImpulseTrackingGoal */
-    class OSIMMOCOIMPULSETRACKINGGOAL_API MocoImpulseTrackingGoalGroup : public Object {
-        OpenSim_DECLARE_CONCRETE_OBJECT(MocoImpulseTrackingGoalGroup, Object);
+    @see MocoContactImpulseTrackingGoal */
+    class OSIMMOCOCONTACTIMPULSETRACKINGGOAL_API MocoContactImpulseTrackingGoalGroup : public Object {
```

Comment:
Since we'll be adding this directly to the OpenSim API, we'll use a different macro here. Replace `OSIMMOCOCONTACTIMPULSETRACKINGGOAL_API` with `OSIMMOCO_API`.